configurations/admin.template.ini

```diff
@@ -1153,6 +1153,12 @@ moduls.loginSSO.permissionName = ALLOW_SSO_PER_USER
 moduls.loginSSO.group = GROUP_LOGIN_SSO_OPTIONS
 moduls.loginSSO.skip = true
 
+moduls.allowThemeEditor.enabled = true
```

Comment:
remove the allow from allowThemeEditor
and in the label change to Enable Theme Editor